api/utils/health_utils.py

```
@@ -205,6 +317,27 @@ def get_mysql_status():
         }
 
 
+def get_database_status():
+    try:
+        # 适配点：SELECT 1 是 MySQL/PostgreSQL/GaussDB 都支持的最小探针。
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)